src/istio/chart/templates/istio-dashboards.yaml

```diff
@@ -0,0 +1,19 @@
+# Copyright 2025 Defense Unicorns
+# SPDX-License-Identifier: AGPL-3.0-or-later OR LicenseRef-Defense-Unicorns-Commercial
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: uds-istio-grafana-dashboards
+  namespace: {{ .Release.Namespace }}
+  labels:
+    grafana_dashboard: "1"
+  annotations:
+    {{- toYaml .Values.dashboardAnnotations | nindent 4 }}
+data:
+  # Load dashboards from files
+  {{- $files := .Files }}
+  {{- range $path, $bytes := .Files.Glob "dashboards/*.json" }}
+  {{ base $path }}: |-
+    {{ $files.Get $path | nindent 4 }}
+  {{- end }}
```

Comment:
I think this is actually the wrong chart to add this to - this is chart used for each gateway, so we'd end up with 2-3x copies of this dashboard (unsure if Grafana deconflicts, but there would be one per gateway namespace).

I think this should be moved to https://github.com/defenseunicorns/uds-core/tree/1623-introduce-keycloak-grafana-dashboards/src/istio/common/chart/templates